docs/UserGuide.md

```diff
@@ -333,20 +333,8 @@ This command creates a boolean template. Similar to the [circuit template](#6-ci
 * The `GATE` can be chosen from the following:
     * `and`, `or`, `xor`, `nand`, `nor`, `xnor`
 
-Example of usage:
-
-`template and`
-
-Expected outcome:
-
-```
-      OUT
-       |
-   B       C
-
-OUT = B AND C
-B = ?
-C = ?
+The diagram below shows an example when using the `template and` command.
+![Simplified](diagrams/TemplateBooleanAnd.png)
 ```
```

Comment:
Broken. Also why not the same format as the other images? The colours and all.
```suggestion

![Simplified](diagrams/TemplateBooleanAnd.png)

```

Reply:
Changed, let me know if it's okay

Reply:
Yep image is ok, but formatting still broken. Commit the suggestion ⬆️